src/components/JdsCalendar/Calendar.vue

```diff
@@ -0,0 +1,189 @@
+      </div>
+    </div>
+    <div class="jds-calendar__list-of-days">
+      <div 
+        v-for="(name,index) in listOfDays" 
+        :key="index" 
+        class="jds-calendar__name-of-day"
+      >
+        {{ name }}
+      </div>
+    </div>
+    <div class="jds-calendar__days">
+      <div 
+        @click="onCalendarClick(date)" 
+        v-for="date in days" 
+        :key="date.id"
+        :class="{
+          'jds-calendar__days-day' : true,
+          'jds-calendar__days-day--selected': isSelected(date),
+          'jds-calendar__days-day--today': isToday(date),
+          'jds-calendar__days-day--holiday': isSunday(date),
+          'jds-calendar__days-day--gray': isNotThisMonth(date),
+        }"
+      >
+        {{ date.date }}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { getDays, listOfDays, listOfMonth } from '../../utils/calendar'
+import { fnDate } from '../../utils/date-input'
+import JdsIcon from '../JdsIcon'
+
+export default {
+  name: 'jds-calendar',
+  components: {
+    JdsIcon
+  },
+  model: {
+    prop: 'value',
+    event: 'change',
+  },
+  props:{
+    /**
+     * The name for calendar
+     */
+    name: {
+      type: String
+    },
+    /**
+     * Type of calendar
+     * Value is `picker | range`
+     */
+    type: {
+     type: String,
+     default: 'picker'
+    },
+    /**
+     * You can set minimal date for
+     * the earliest acceptable date
+     * The value of the min date format follows the pattern `DD/MM/YYYY`.
+     */
+    min: {
+      type: String,
+      default: '01/01/1900'
+    },
+    /**
+     * You can set maximal date for
+     * the latest acceptable date
+     * The value of the max date format follows the pattern `DD/MM/YYYY`.
+     */
+    max: {
+      type: String,
+      default: '31/12/9999'
+    },
+    /**
+     * Bound model.
+     * @name value
+     * @model
+     * The value of the input date format follows the pattern `DD/MM/YYYY`.
+     * if value not set the value will be current date
+     */
+    value: {
+      type: String,
+    },
+  },
+  data() {
+    return {
+      mValue: undefined,
+      selectedValue: null,
+      listOfDays,
+      listOfMonth
+    }
+  },
+  watch: {
+    value: {
+      handler: function (v) {
+        if(typeof v === 'undefined' || v === null || !this.isDateStringValid(v)){
+          this.mValue = new Date()
+        }else{
+          this.mValue = fnDate.parseDate(v)
+          this.selectedValue = v.split('/')
+        }
+      },
+      immediate: true,
+    }
+  },
+  computed: {
+    days () {
+      return getDays(this.mValue)
+    }
+  },
+  methods: {
+    isDateStringValid(dateString){
+      let valid = true
+      const arrayOfDate = dateString.split("/")
+      const date = new Date(arrayOfDate[1]+"/"+arrayOfDate[0]+"/"+arrayOfDate[2])
+      if(date.toString() === 'Invalid Date'){
+        valid = false
+      }
+      return valid
```

Comment:
It seems like this lines were the causes of issue mentioned by Shinta.

Due to Storybook Controls behavior, in which `value` is typed sequentially, there is a case such as this.
1) I want to entry '08/02/2000' as `DateInput#value`
2) I type each character into Storybook Controls
3) `isDateStringValid` will return false for each typed character, unless it reach `08/02/`, in which it return true

OTOH, `parseDate` function can't correctly parse `08/02/` as valid date.

Reply:
This error can be ignored if `value` is not designed to accept string sequentially.